java/lang/Integer/IntegerTest.cpp

```diff
@@ -308,16 +308,24 @@ TEST (JavaLang, IntegerLongValue) {
 
 TEST (JavaLang, IntegerFloatValue) {
     // Input different values of type int to compare to the realResult of Integer::floatValue
+
+    // Make a comparison between expected realResult and the real result
     Integer positiveInteger = 12345;
     float expectedResult = 12345.00;
     float realResult = positiveInteger.floatValue();
     ASSERT_TRUE(expectedResult == realResult);
 
-
+    // Make a comparison between expected realResult and the real result
     Integer negativeInteger = -12345;
     expectedResult = -12345.00;
     realResult = negativeInteger.floatValue();
     ASSERT_TRUE(expectedResult == realResult);
+
+    // Make a comparison between the not expected realResult and the real result
+    Integer notExpectedInteger = -12345;

```

Comment:
for that code : ASSERT_FALSE(notExpectedResult == realResult);  
I want to test the case : the realResult and the expectedResult are different. So I call it notExpectedInteger.